pkg/adapter/v2/cloudevents_test.go

```diff
@@ -162,7 +162,7 @@ func validateSent(t *testing.T, ce *test.TestCloudEventsClient, want string) {
 
 func validateNotSent(t *testing.T, ce *test.TestCloudEventsClient) {
 	if got := len(ce.Sent()); got != 0 {
-		t.Errorf("Expected 0 event to be sent, got %d", got)
+		t.Error("Expected 0 event to be sent, got", got)
```

Comment:
nit: use correct grammar
```suggestion
		t.Error("Expected 0 events to be sent, got", got)
```

Reply:
I think I fixed a few others like this.
Done